Linux/sgx/Makefile

```diff
@@ -72,7 +72,6 @@ endif
 endif
 
 $(PACKAGE_LIB)/$(OPENSSL_LIB):
-	chmod 755 $(BUILD_OPENSSL)
```

Comment:
Why remove this chmod command, is it intentional?

Reply:
It seems unnecessary, if the code was checked out directly checked out , or decompressed from the code tar.gz package in Linux environment.

Reply:
> Why remove this chmod command, is it intentional?

the code was auto-merged from the master branch. Let me check the details. It is under testing.

Reply:
> Why remove this chmod command, is it intentional?

confirmed with Jing. That command was intentionally removed in the master branch sometime ago.

Reply:
test passed. @jinghe-INTC @ShundaZhang